src/database/repositories/irepository.py

```diff
@@ -15,7 +15,7 @@ class IRepository(abc.ABC, Generic[T]):
 
     def __init__(self):
         self.engine = EngineHolder().get_engine()
-        self.session = create_session(self.engine)
+        self.session = scoped_session(sessionmaker(bind=self.engine))()
```

Comment:
Why bind session again

Reply:
sessionmaker is a factory function create class Session. i already bind engine in database/orm.py

![image](https://github.com/user-attachments/assets/f034d90b-669c-453d-bf07-7ec64de2fc3f)